src/test/java/uk/gov/justice/probation/courtcaseservice/controller/CourtCaseControllerPutByHearingIdIntTest.java

```diff
@@ -114,6 +129,19 @@ void beforeEach() throws Exception {
                 .build());
     }
 
+    @Test
```

Comment:
should probably do this in a separate class really but initially I thought I would use RestAssured. Unfortunately I could not see a way to make requests to different URLs (well, different ports)

Reply:
https://github.com/rest-assured/rest-assured/issues/534